mlir/lib/Dialect/Rock/Transforms/AffixTuningParameters.cpp

```diff
@@ -105,7 +105,8 @@ void AffixTuningParameters::affixTuningParametersImpl(
     perfConfig = perfConfigAttr.getValue().str();
   }
   GemmFeatures features = op.getGemmFeatures();
-  if (bitEnumContainsAll(features, GemmFeatures::mfma)) {
+  if (bitEnumContainsAll(features, GemmFeatures::mfma) ||
+      bitEnumContainsAll(features, GemmFeatures::wmma)) {
```

Comment:
You will continue to review the tuning details after this PR, right? I don't quite expect the tuning parameter derive logic to be all compatible.

Reply:
Yes, after this PR I will open another one about tuning